.github/dependabot.yml

```diff
@@ -8,6 +8,7 @@ updates:
       timezone: Europe/Berlin
     ignore:
       - dependency-name: devtools-protocol
+      - dependency-name: '@types/node'
```

Comment:
Add rollup aswell as v4 requires Node 18

Reply:
For `rollup`, we should can do a temporary ignore on the PR. We _always_ want to ignore @types/node